client-index/Cargo.toml

```diff
@@ -8,10 +8,11 @@ edition = "2018"
 chain-core = { path = "../chain-core" }
 client-common = { path = "../client-common" }
 failure = "0.1"
-rlp = "0.3"
-bincode = "1.1"
+parity-codec = "3.5.1"
+parity-codec-derive = "3.3.0"
```

Comment:
Remove `parity-codec-derive` dependency and add `"derive"` feature in `parity-codec`.